chat.revolt.RevoltDesktop.yaml

```diff
@@ -1,16 +1,13 @@
 app-id: chat.revolt.RevoltDesktop
 runtime: org.freedesktop.Platform
-runtime-version: '21.08'
+runtime-version: '22.08'
 sdk: org.freedesktop.Sdk
-
 base: org.electronjs.Electron2.BaseApp
-base-version: '21.08'
-
+base-version: '22.08'
 sdk-extensions:
 - org.freedesktop.Sdk.Extension.node16
-
+command: run.sh
```

Comment:
Why is this script renamed and added as a source file (not as a script like it was before)?

Reply:
it was a test and now its back to revolt-desktop.sh